symbolic-debuginfo/src/lib.rs

```diff
@@ -39,6 +39,7 @@ mod base;
 mod object;
 mod private;
 
+pub mod bcsymbolmap;
```

Comment:
Since BCSymbolMaps are tied to `MachO`, could we just move it over? That way you don't have to import from two modules when dealing with those files, and they will all be documented together. We could also add a mention to them in the `macho` module doc comment.